parquet-avro/src/test/java/org/apache/parquet/avro/TestArrayCompatibility.java

```diff
@@ -603,7 +604,7 @@ public void testAvroCompatOptionalGroupInList() throws Exception {
   public void testAvroCompatOptionalGroupInListWithSchema() throws Exception {
     Path test = writeDirect(
         "message AvroCompatOptionalGroupInListWithSchema {" + "  optional group locations (LIST) {"
-            + "    repeated group array {"
+            + "    repeated group my_list {"
```

Comment:
This test started failing after I made the change to AvroRecordConverter. Looking at it, this schema didn't match the logical type spec for List types -- any repeated group named `array` must contain element type fields directly. So I updated the schema naming.

Reply:
I agree with you. Is it better to add a test case with the incorrect schema and check it throws?

Reply:
sounds good! added.